irc.html

```diff
@@ -10,24 +10,30 @@ <h1>Channels</h1>
 <div>
   <p>
     Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
+    Please note that in order to chat on #minetest-hub you need to
+    ask an operator for voice and to chat on #minetest-dev you 
```

Comment:
Long sentence, perhaps:
`ask an operator for voice, to chat on #minetest-dev you`
?